test/utils.py

```diff
@@ -694,7 +694,7 @@ def start(self):
     args = environment.binary_args('vtctld') + [
             '-debug',
             '-templates', environment.vttop + '/go/cmd/vtctld/templates',
-            '-schema-editor-dir', environment.vttop + '/go/vt/vtgate',
+            '-schema-editor-dir', environment.vttop + '/go/cmd/vtctld',
```

Comment:
Correct, in fact, another coming pull request will merge schema_editor dir and templates dir. In other words, templates dir will contain all UI related stuff.